api/src/main/java/module-info.java

```diff
@@ -143,7 +155,8 @@
  *     &#64;Query("SELECT o FROM Order o WHERE o.address.zipCode=?1")
  *     List&lt;Order&gt; forZipCode(int zipCode);
  *
- *     void save(Order order);
+ *     &#64;Save
+ *     Order save(Order order);
```

Comment:
```suggestion
 *     Order checkout(Order order);
```

How about this? I used Amazon as sample, we checkout an order.

Reply:
Good suggestion. I added it.